syntax = "proto3";

package openauto.btservice.proto;

message PhoneResponse
{
    int32 status_code = 1;
}
